### PR TITLE
Create leap_year.c

### PR DESCRIPTION
**Pull request by born2bdev**
Added code to find leap year between two years in C Language

### CODE REVIEW

**Comment by Vaibhav1671998** [comment]
@arnavb Have a look at this one

File: code/unclassified/leap_year/leap_year.c [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+
+bool checkLeapYear(int year)
+{
+    if (year % 400 == 0)
```
Comment (arnavb) [inline code/unclassified/leap_year/leap_year.c]: The whole thing is one line:

```C
return (year % 400 == 0) || ((year % 100 != 0) && (year % 100 == 4));
```

File: code/unclassified/leap_year/leap_year.c [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+int main()
+{
+    int year1 , year2;
+    printf("Enter the first year: ");
+    scanf("%d",&year1);
+    printf("Enter the second year year: ");
+    scanf("%d",&year2);
+    printf("Leap years between %d and %d are : \n",year1,year2);
```
Comment (arnavb) [inline code/unclassified/leap_year/leap_year.c]: The leap years

**Comment by Vaibhav1671998** [comment]
I hadn't made this change @arnavb 
return (year % 400 == 0) || ((year % 100 != 0) && (year % 100 == 4));

because i think the one i had written is much more clearer

**Comment by arnavb** [comment]
The change your are referring to is the most common implementation of leap year, which I was suggesting.

**Comment by Vaibhav1671998** [comment]
It's might be most common but i think mine was much easier to understand .

**Comment by Vaibhav1671998** [comment]
@AdiChat Merge this if u r happy with this :)

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**